tools/cli/adminTimers.go

```diff
@@ -36,18 +38,97 @@ import (
 	"github.com/uber/cadence/common/quotas"
 )
 
+type Loader interface {
+	Load() []*persistence.TimerTaskInfo
+}
+
+type Printer interface {
+	Print(timers []*persistence.TimerTaskInfo) error
+}
+type TimerPrinter struct {
```

Comment:
Can we call it something else? Initial glance suggests that it might implement `Printer` interface. While it is an aggregate structure of both loader and printer.